<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGB Desk</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="All">
     <item>
      <layout class="QHBoxLayout" name="RGBChose">
       <item>
        <layout class="QVBoxLayout" name="Sliders">
         <item>
          <widget class="QSlider" name="sRed">
           <property name="maximum">
            <number>256</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sGreen">
           <property name="maximum">
            <number>256</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sBlue">
           <property name="maximum">
            <number>256</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Names">
         <item>
          <widget class="QLabel" name="lRed">
           <property name="text">
            <string>Red:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lGreen">
           <property name="text">
            <string>Green:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lBlue">
           <property name="text">
            <string>Blue:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Values">
         <item>
          <widget class="QLabel" name="lRedVal">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lGreenVal">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lBlueVal">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Connect">
       <item>
        <widget class="QLabel" name="lSelectPort">
         <property name="text">
          <string>Wybierz port:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cPort">
         <item>
          <property name="text">
           <string>COM 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM 3</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bConnect">
         <property name="text">
          <string>Połącz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Upload">
       <item>
        <widget class="QPushButton" name="bAutoUpload">
         <property name="text">
          <string>Automatycznie ustawiaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bManualUpload">
         <property name="text">
          <string>Ustaw ręcznie</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpcje">
    <property name="title">
     <string>Opcje</string>
    </property>
    <addaction name="actionUstawienia"/>
    <addaction name="actionWyj_cie"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionPomoc_techniczna"/>
    <addaction name="actionO_programie"/>
    <addaction name="actionO_autorze"/>
   </widget>
   <addaction name="menuOpcje"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionUstawienia">
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
  <action name="actionWyj_cie">
   <property name="text">
    <string>Wyjście</string>
   </property>
  </action>
  <action name="actionPomoc_techniczna">
   <property name="text">
    <string>Pomoc techniczna</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionO_autorze">
   <property name="text">
    <string>O autorze</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sRed</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lRedVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sGreen</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lGreenVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sBlue</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lBlueVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
